src/pages/BusinessSpecifics/BusinessSpecifics.tsx

```diff
@@ -2,14 +2,35 @@ import { FormControl, TextField, Button, MenuItem, Select } from "@mui/material"
 import { useState } from "react"
 import { currencies } from "./currencies"
 
-export const BusinessSpecifics = () => {
 
-    const [BusinessName, setBusinessName] = useState("")
-    const [LocationName, setLocationName] = useState("")
+export const BusinessSpecifics = () => {
+    const url = 'https://invent0ry-back3nd.herokuapp.com/businesses/'
+    const [businessName, setBusinessName] = useState("")
+    const [locationName, setLocationName] = useState("")
     const [currency, setCurrency] = useState(currencies.USD)
 
     const currencyList = Object.keys(currencies).map(key => (<MenuItem value={key}>{key}</MenuItem>))
 
+    function handleSubmit(){
+        const options = {

```

Comment:
all this logic needs to be separated into the service directory

Reply:
ahh, I see. Do all of the api requests go in here?